InsiteTeamTask/Repositories/DataRepository.cs

```diff
@@ -1,9 +1,7 @@
-﻿using InsiteTeamTask.MockData;
-using InsiteTeamTask.Models;
+﻿using InsiteTeamTask.Models;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 
 namespace InsiteTeamTask.Repositories
 {
```

Comment:
Would have been nice to see `IDataService` injected into this class, rather than creating a new one manually with each method call